### DISCUSSION TITLE
Windows Support

### DISCUSSION THREAD

**Discussion by jamalsa** [supabase-community/postgres-language-server#26] [Ideas]
Is there any plan to support Windows in the future? I think the blocker would be [libpg_query](https://github.com/pganalyze/libpg_query) which currently doesn't support Window (https://github.com/pganalyze/libpg_query/issues/44). 

If Supabase can help fix this issue, it will not only benefit this project but also many libraries which use libpg_query.

**Comment by psteinroe** [comment]
there has been [a thread on hacker news](https://news.ycombinator.com/item?id=37023150) about this.

summary: we know about this limitations, and are happy to look into it at the appropriate time. for now, our focus is the language server. windows dev can host the server and connect to it via tcp in the meanwhile.
